nakama/match.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+
+	"github.com/heroiclabs/nakama-common/runtime"
+)
+
+type ReceiptMatch struct{}
+
```

Comment:
we should add an iface guard for the impl `var _ runtime.Match = &ReceiptMatch{}`

Reply:
done

Reply:
done